include/ofi_util.h

```diff
@@ -580,8 +580,8 @@ ssize_t ofi_cq_read_entries(struct util_cq *cq, void *buf, size_t count,
 	for (i = 0; i < (ssize_t) count; i++) {
 		entry = ofi_cirque_head(cq->cirq);
 		if (!(entry->flags & UTIL_FLAG_AUX)) {
-			if (src_addr && cq->src)
-				src_addr[i] = cq->src[ofi_cirque_rindex(cq->cirq)];
+			if (src_addr)
+				src_addr[i] = cq->src ? cq->src[ofi_cirque_rindex(cq->cirq)] : FI_ADDR_NOTAVAIL;
```

Comment:
line length is getting long.  Also, see line 599, which I think needs a similar (but not exactly the same) change.